sdk/core/azure-core/src/main/java/module-info.java

```diff
@@ -28,6 +31,9 @@
     exports com.azure.core.util.serializer;
     exports com.azure.core.util.tracing;
     exports com.azure.core.cryptography;
+    exports com.azure.core.util.serializer.jsonwrapper;
+    exports com.azure.core.util.serializer.jsonwrapper.jacksonwrapper;
+    exports com.azure.core.util.serializer.jsonwrapper.api;
```

Comment:
Do these exports get used in any public APIs? I don't see them being used in any client code.

Reply:
jsonwrapper was created to be a utility for json deserializartion, and is intended to be publicly used. Currently being used in azure-search sdk code in tests and samples. See https://github.com/Azure/azure-sdk-for-java/pull/5853

Reply:
So the expectation, as shown in one of the samples, is that a consumer of the SDK will create a jsonwrapper object to deserialize the JSON blobs returned to them?

Reply:
in this case yes, the end user can use it to deserialize the documents into well known types, since the current implemetation of azure-core does not support cusm deserialization currently. see https://github.com/Azure/azure-sdk-for-java/issues/5710

Reply:
@alzimmermsft I've reverted the changes to jsonwrapper. It is moved back under implemetation and azure-search is not using it. This will allow to focus this PR on azure-search SDK and handle jsonwrapper seperatly. see commit https://github.com/Azure/azure-sdk-for-java/pull/7064/commits/41b83d9fefa98d1bd2de6c926b361776b57fe616